pkg/synopsysctl/cmd_destroy.go

```diff
@@ -89,28 +95,35 @@ func destroy(namespace string) error {
 		if crdNames, ok := cmData["CrdNames"]; ok {
 			crds = strings.Split(crdNames.(string), ",")
 		}
+		isExist = true
 	}
 
 	// delete namespace
-	checkErr := util.CheckResourceNamespace(restconfig, kubeClient, strings.Join(crds, ","), namespace, true)
-	if err != nil && isForceDestroy {
-		log.Warnf("%s. namespace cannot be deleted", err.Error())
-	} else if err != nil && !isForceDestroy {
-		return fmt.Errorf("%s. It is not recommended to destroy the Synopsys Operator so these resources can continue to be managed. If you are sure you want to delete the Synopsys Operator anyway then you can use the 'force' option which will keep all the instances and delete only the Synopsys Operator", err.Error())
-	}
+	if isExist {
+		isExist, err := util.CheckResourceNamespace(restconfig, kubeClient, strings.Join(crds, ","), namespace, true)
+		if isExist && err != nil && isForceDestroy {
+			log.Warnf("%s. namespace cannot be deleted", err.Error())
+		} else if isExist && err != nil && !isForceDestroy {
+			return fmt.Errorf("%s. It is not recommended to destroy the Synopsys Operator so these resources can continue to be managed. If you are sure you want to delete the Synopsys Operator anyway then you can use the 'force' option which will keep all the instances and delete only the Synopsys Operator", err.Error())
+		} else if err != nil {
+			log.Error(err)
```

Comment:
should return err

Reply:
just proceeding down to delete the cluster roles, role bindings